eng/pipelines/azure-pipelines.yml

```diff
@@ -63,169 +91,137 @@ resources:
-
-          variables:
-            - name: _buildScript
-              value: $(Build.SourcesDirectory)/build.sh --ci
-
-          preSteps:
-            - checkout: self
-              fetchDepth: 1
-              clean: true
-
-          steps:
-          - template: /eng/pipelines/templates/BuildAndTest.yml
-            parameters:
-              runAsPublic: ${{ eq(variables._RunAsPublic, True) }}
-              dotnetScript: $(Build.SourcesDirectory)/dotnet.sh
-              buildScript: $(_buildScript)
-              buildConfig: $(_BuildConfig)
-              repoLogPath: $(Build.Arcade.LogsPath)
-              repoTestResultsPath: $(Build.Arcade.TestResultsPath)
-              isWindows: false
-
-
-# ----------------------------------------------------------------
-# This stage performs quality gates checks
-# ----------------------------------------------------------------
-- stage: codecoverage
-  displayName: CodeCoverage
-  dependsOn:
-    - build
-  condition: succeeded('build')
-  variables:
-  - template: /eng/common/templates/variables/pool-providers.yml
-  jobs:
-  - template: /eng/common/templates/jobs/jobs.yml
-    parameters:
-      enableMicrobuild: true
-      enableTelemetry: true
-      runAsPublic: false
-      workspace:
-        clean: all
-
-      # ----------------------------------------------------------------
-      # This stage downloads the code coverage reports from the build jobs,
-      # merges those and validates the combined test coverage.
-      # ----------------------------------------------------------------
+  repositories:
+  - repository: 1ESPipelineTemplates
+    type: git
+    name: 1ESPipelineTemplates/1ESPipelineTemplates
+    ref: refs/tags/release
+
+extends:
+  template: v1/1ES.Official.PipelineTemplate.yml@1ESPipelineTemplates
+  parameters:
+    sdl:
+      sourceAnalysisPool:
+        name: NetCore1ESPool-Internal
+        image: windows.vs2022preview.amd64
+        os: windows
+    customBuildTags:
+    - ES365AIMigrationTooling
+
+    stages:
+
+    # ----------------------------------------------------------------
+    # This stage performs build, test, packaging
+    # ----------------------------------------------------------------
+    - stage: build
+      displayName: Build
       jobs:
-      - job: CodeCoverageReport
-        timeoutInMinutes: 10
-
-        pool:
-          ${{ if eq(variables['System.TeamProject'], 'public') }}:
-            name: $(DncEngPublicBuildPool)
-            demands: ImageOverride -equals build.ubuntu.2004.amd64.open
-          ${{ else }}:
-            name: $(DncEngInternalBuildPool)
-            demands: ImageOverride -equals build.ubuntu.2004.amd64
-
-        preSteps:
-          - checkout: self
-            fetchDepth: 1
-            clean: true
-
-        steps:
-        - script: $(Build.SourcesDirectory)/build.sh --ci --restore
-          displayName: Init toolset
-
-        - template: /eng/pipelines/templates/VerifyCoverageReport.yml
```

Comment:
@davidfowl @eerhardt FYI we're currently dropping the CC report from the internal pipeline. Cobertura files are still being collected and published as artifacts, however, those won't get aggregated or turned into an HTML report. 
This functionality we can bring back later (if required/subject to prioritisation).

Reply:
Honestly, I'm not sure we are even using/looking at code coverage from our PR validation builds. At least I haven't been.